advanced_samples/Bulk_Upload/SampleCollections/DataviewWind/appsettings.json

```diff
@@ -0,0 +1,11 @@
+﻿{
```

Comment:
This should be renamed to `appsettings.placeholder.json` and this file should be deleted. Make sure it can't be added back afterwards, and make sure the README mentions to copy/rename the file (that text has been added to all the other READMEs)